nowcasting_dataset/config/load.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import gcsfs
+import os
+import io
+import yaml
+from nowcasting_dataset.config.model import Configuration
+
+logger = logging.getLogger(__name__)
```

Comment:
We should maybe use the logger name `nowcasting_dataset`?  (Or does `__name__` resolve to `nowcasting_dataset`?!?)

Reply:
Looks like __name__ essentially resolves it self to the file name i.e 'nowcasting_dataset.config.load'. I think this means then you can 'more' easily turn things on and off at a more granular level, if needed. It also has the bonus of if the file names change, you don't have to up date it.

This has a bit on it at the bottom
https://www.machinelearningplus.com/python/python-logging-guide/

This seems to say to do it too
https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/#:~:text=Use%20__name__%20as%20the%20logger%20name&text=as%20long%20as%20their%20logging,to%20output%20the%20logger%20name.

In a quick search in google, i cant seem to find good reasons not to do this

Reply:
Its funny, I just do that by habit, and I've never been asked why before

Reply:
Cool beans, I've learnt something new, sounds like I should change to using `getLogger(__name__)`!  Thanks for looking into it!